src/formio/components/address.ts

```diff
@@ -0,0 +1,22 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export interface AddressData {
+  postcode: string;
+  housenumber: string;
+  houseletter?: string;
+  housenumberaddition?: string;
```

Comment:
```suggestion
  postcode: string;
  houseNumber: string;
  houseLetter?: string;
  houseNumberAddition?: string;
```
should we make the keys optional or just expect/initialize to empty strings for optional fields?